### PR TITLE
RF-14256 Set priority on RFML

### PR DESCRIPTION
**Pull request by fuzzylita**
/reviewme @rainforestapp/backend @epaulet 

Did my best but honestly, serious copypasta. Please let me know if I biffed anything ❤️

### CODE REVIEW

File: rainforest/rfml.go [hunk 1]
```diff
@@ -140,6 +140,14 @@ func (r *RFMLReader) ReadAll() (*RFTest, error) {
 					parsedRFTest.FeatureID = FeatureIDInt(featureID)
 				case "state":
 					parsedRFTest.State = value
+				case "priority":
+					value = strings.ToUpper(value)
+					switch value {
+					case "P1", "P2", "P3", "":
+						parsedRFTest.Priority = value
+					default:
+						return parsedRFTest, &parseError{lineNumStr, "Redirect value must be one of P1, P2, P3"}
```
Comment (igorfrenkel) [inline rainforest/rfml.go]: The error says only p1, p2, p3 acceptable, but in the case statement on line 146 you also accept "". Is this intentional?

Comment (fuzzylita) [reply rainforest/rfml.go]: I thought I should, but was working with a seasoned Go dev to learn while doing and he suggested that it was unnecessary 😅

Happy to add!

Comment (epaulet) [reply rainforest/rfml.go]: Well the issue is that we need a way to "unset" priority, right? Seems like empty string would be the way to go. Does the API remove priority if you send an empty string?

Comment (fuzzylita) [reply rainforest/rfml.go]: I'll add that in a separate PR.

**Comment by fuzzylita** [comment]
@jbarber priority is optional, so it should be fine.

**Pull request merged**